index.html

```diff
@@ -36,3 +34,155 @@ <h3 class="intro">I'm Bunbee<br>Glad to see<br> you!</h3>
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(3).png" alt="firsproject">
+        </div>
+        <h2 class="card-header">Tonic</h2>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+    <section class="card-section">
+        <div class="card-image">
+            <img src="img/SnapshootPortfolio(4).png" alt="firstoject">
+        </div>
+        <h2 class="card-header">Multi-Post<br>Stories</h2>
+        <ul class="card-links">
+            <li>CANOPY</li>
+            <li> Black End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p class="card-text">A daily selection of privately <br>personalised reads; no accounts or sign-ups required</p>
+        <div class="lang-section">
+           <ul class="card-lang">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+           </ul>
+        </div>
+        <button class="btn1">See Project</button>
+    </section>
+
+
+    <section class="about-me">
+        <h1 class="about-header">About<br> Myself</h1>
+        <p class="about-para"> Hello I'm a software devoloper! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+        <p class="connect">LET'S CONNECT</p>
+        <ul class="social-media">
+            <li class="fa fa-twitter fa-2x"></li>
+            <li class="fa fa-linkedin fa-2x"></li>
+            <li class="fa fa-medium fa-2x"></li>
+            <li class="fa fa-github fa-2x"></li>
+            <li class="fa fa-facebook fa-2x"></li>
+           </ul>
+        <button class=" btn1 btn2">Get my resume</button>
+
+        <ul class="tools">
+            <li>
+                <div class="lang-arrow">
+                    <h3>Languages</h3>
+                    <i class="fa fa-angle-down fa-2x la"></i>
+                </div>
+                
+                <ul class=langua>
+                    <li class="lang-item">
+                        <img src="img/Ellipse1.png" alt="lang">
+                        <p>JavaScript</p>
+                    </li>
+                    <li class="lang-item">
+                        <img src="img/Ellipse1(1).png" alt="lang">
+                        <p>HTML</p>
+                    </li>
+                    <li class="lang-item">
+                        <img src="img/Ellipse1(2).png" alt="lang"> 
+                        <p>CSS</p> 
+                    </li>
```

Comment:
- [ ] [OPTIONAL] I think it could be a good idea to improve the layout of the languages categories because for now, it seems to be a bit different from the Figma design, see the difference below 👇 
- Your design
![image](https://user-images.githubusercontent.com/55465133/130331652-d013e85d-88b8-4395-bfb5-ad1ec3160fa4.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130331681-88aa6b18-aeaa-4431-a44f-0d9d5c34b1ad.png)

It could be nicer to adjust the layout to make it look like the Figma design 😉

Reply:
is it ok if i remove the list items as we cannot use javascript implementation

Reply:
Yeah actually just try to stick to the design in Figma please